extensions/amp-3q-player/0.1/test/test-amp-3q-player.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
@aghassemi I'm currently implementing the test. 
In the test, the amp-3q-player.js is receiving the play,pause .. requests, but it's not forwarding it to the player iframe. this.iframe_ is not defined if the video player test is running. Maybe you can help.